test/goaway_test.go

```diff
@@ -816,8 +801,17 @@ func (s) TestClientSendsAGoAway(t *testing.T) {
 			}
 		}
 	}()
+
+	cc, err := grpc.Dial(lis.Addr().String(), grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
nit: Shall we use NewClient since Dial is in deprecation stage?

Reply:
+1

Reply:
Updated.